style.css

```diff
@@ -0,0 +1,806 @@
+*,
+*::before,
+*::after {
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Roboto', sans-serif;
+  font-size: inherit;
+  border-bottom-color: #000;
+}
+
+button {
+  border: none;
+  background: transparent;
+  right: inherit;
+}
+
+ul {
+  list-style-type: none;
+}
+
+a {
+  text-decoration: none;
+  color: inherit;
+}
+
+html {
+  font-size: 10px;
+}
+
+.main-header {
+  position: fixed;
+  z-index: 100;
+  width: 100%;
+}
+
+.icon {
+  display: block;
+  background-position: center;
+  background-size: contain;
+  background-repeat: no-repeat;
+  padding: 11px;
+  background-color: rgb(47, 47, 47);
```

Comment:
Not every icon has a dark grey and border-radius rounded style. This is also affecting the navbar icons. Please apply this style only to the pertinent icons.
![Screenshot_2020-03-22 The Next Web](https://user-images.githubusercontent.com/16193212/77241601-3b934480-6bd3-11ea-9cdb-23f20c56e056.png)

![Screenshot_2020-03-22 TNW - International technology news, business and culture](https://user-images.githubusercontent.com/16193212/77241600-3afaae00-6bd3-11ea-8efc-6c533ad3bc15.png)